src/coreclr/System.Private.CoreLib/src/System/RuntimeHandles.cs

```diff
@@ -304,7 +304,28 @@ internal RuntimeType GetRuntimeType()
         }
 
         [MethodImpl(MethodImplOptions.InternalCall)]
-        internal static extern CorElementType GetCorElementType(RuntimeType type);
+        private static extern CorElementType GetCorElementTypeInternal(RuntimeType type);
+
+        internal static CorElementType GetCorElementType(RuntimeType type)
+        {
+            if (type is null)
+            {
+                throw new ArgumentNullException(SR.Arg_InvalidHandle);
+            }
+            return GetCorElementTypeInternal(type);
+        }
+
+        [MethodImpl(MethodImplOptions.InternalCall)]
+        private static extern CorElementType GetVerifierCorElementTypeInternal(RuntimeType type);
+
+        internal static CorElementType GetVerifierCorElementType(RuntimeType type)
```

Comment:
Is `RuntimeType.GetUnderlyingType` doing the same thing?

Reply:
(`GetUnderlyingCorElementType` would be a better name for the existing method.)

Reply:
Yes, it is. I will update to that.